docs/user-manual/en/copied-message-properties.md

```diff
@@ -0,0 +1,37 @@
+# Properties for Copied Messages
+
+There are several operations within the broker that result in copying a
+message. These include:
+
+- Diverting a message from one address to another.
+- Moving an expired message from a queue to the configured `expiry-address`
+- Moving a message which has exceeded its `max-delivery-attempts` from a queue
+  to the configured `dead-letter-address`
+- Using the management API to administratively move messages from one queue to
+  another
+
+When this happens the body and properties of the original message are copied to
+a new message. However, the copying process removes some potentially important
+pieces of data so those are preserved in the following special message
+properties:
+
+- `_AMQ_ORIG_ADDRESS`
```

Comment:
These are also documented in AMQP. 

Perhaps you could merge the content of that section here, and add a note about message annotations for AMQP messages?

Reply:
They're referenced in the AMQP documentation but only insofar as how they are translated. I think the AMQP doc is where that should be.

Reply:
Can you just a link then, stating there are some small differences with AMQP messages perhaps?